packages/oncoprintjs/src/js/oncoprintwebglcellview.ts

```diff
@@ -1898,6 +2065,34 @@ export default class OncoprintWebGLCellView {
                 track_id,
                 false
             );
+
+            const custom = model.getTrackCustomOptions(track_id);
+            if (gapOffsets[0]) {
+                const gaps = _.isEmpty(model.ids_after_a_gap.get())
+                    ? undefined
+                    : custom.find(t => !!t.gapLabelsFn)?.gapLabelsFn(model);
```

Comment:
duplicate code segment. should we define a `getGaps` function? we can probably include this one as well `const custom = model.getTrackCustomOptions(track_id);`

Reply:
good idea